back/config/tenant_templates/base.yml

```diff
@@ -1,4 +1,23 @@
 models:
+  home_page:
+    -
+      top_info_section_enabled: false
+      top_info_section_multiloc: {}
+      bottom_info_section_enabled: false
+      bottom_info_section_multiloc: {}
+      events_enabled: false
+      projects_enabled: true
+      projects_header_multiloc: {}
+      banner_avatars_enabled: true
+      banner_enabled: true
+      banner_layout: "full_width_banner_layout"
+      banner_signed_in_header_multiloc: {}
+      cta_signed_in_text_multiloc: {}
+      cta_signed_in_type: "no_button"
+      banner_signed_out_header_multiloc: {}
+      banner_signed_out_subheader_multiloc: {}
+      cta_signed_out_text_multiloc: {}
+      cta_signed_out_type: "sign_up_button"
```

Comment:
We actually don't need to specify all values, as is done here.

The problem is that we iterate over these values with `each`, meaning that we could include any single value and this will still create exactly the same record (using default values, and NULL values), but will fail if no value(s) provided, with
```
Loading of the template failed.
undefined method `each' for nil:NilClass
```
So, I decided if we need to specify at least one value, we may as well be explicit and specify all that are set (not NULL).
Downside of this approach is that if we change the default value of any field, it will need updating here and in `e2e` template.

Reply:
Didn't you try to fix it in the script itself? E.g. do not run `each` if the value is `nil`?
Probably, it would be an easy fix and it won't have any downsides.

Reply:
> I would prefer to instead fix the place where `each` is called (do you have the stack trace?), so that templates work with only default values.

I agree with this idea. Will investigate. No stack trace at this point, no, as is just an error message reported in AdminHQ.

Reply:
Looks like it's [here](https://github.com/CitizenLabDotCo/citizenlab-ee/blob/5238cc57f567282d814a328f3086b5947bb9690d/back/engines/ee/multi_tenancy/app/services/multi_tenancy/tenant_template_service.rb#L28) in `tenant_templates_service.rb`. Will experiment.